client/app/worlds/worlds.controller.js

```diff
@@ -16,7 +18,10 @@ angular.module('farseerApp')
     }
 
     $scope.convertToLowerCaseNoSpaces = function(nameToConvert) {
-      return nameToConvert.replace(/ /g, '').toLowerCase();
+      if (nameToConvert !== null && nameToConvert !== undefined) {
```

Comment:
Is there a built-in function that does this...or could it be extracted into your own method isNullOrUndefined(...), returning a boolean?